src/main/webapp/level2.html

```diff
@@ -9,6 +9,7 @@
     <link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
 </head>
 <body>
-    <p>"What animal do you not want to play cards with? (TIP: Upload a picture of the animal for your answer. Submit the file and then submit in the answer box.)"</p>
+    <p>What animal do you not want to play cards with?</p>
```

Comment:
can you have the 'instructions' in normal format and the actuall riddle in the card? So have the line starting with 'Upload a picture' above the riddle 'what animal do you not want to play cards with?' (this sentence as the card)

Reply:
Switched them